python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -103,7 +132,7 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
 def transform_file(
```

Comment:
Do we even need a separate `transform_file` decorator? or can we have the following too instead?

```
        table_from_query = aql.transform(
            file_path="tests/sql/operators/transform/test.sql",
            parameters={"input_table": imdb_table},
            op_kwargs={"output_table": target_table},
        )
```

Reply:
I'd honestly love this solution if we can get it to work. Keeps the API simple. Eventually we could even have the transform function to point at directories to do the live rendering :).

Reply:
> Do we even need a separate transform_file decorator?

The `transform_file` is a function - not a decorator.

> or can we have the following too instead?

No, because `aql.transform` already exists as a decorator in the module and we want to call a function. That is why I previously had created a `transform_sql` which you can use to pass either sql or a file_path, but I reverted it because of discussion in https://github.com/astronomer/astro-sdk/pull/897#discussion_r984080333.

If you only want to have the same name, we can have it in a separate module, but `aql.transform` won't work.